src/EnumSet.php

```diff
@@ -198,11 +202,76 @@ public function count()
         $cnt = 0;
         $ord = 0;
         do {
-            if ($this->bitset & (1 << $ord++)) {
+            if ($this->getBit($ord++)) {
                 ++$cnt;
             }
         } while ($ord !== $this->ordinalMax);
 
         return $cnt;
     }
+
+    /**
+     * Get the bitset
+     * @return int
+     */
+    public function getBitset() {
+        $bitset="";
+        foreach($this->bitset as $int) {
+            $bitset.=pack(
+                $this->integerSize==32?'V':'P',$int);
+        }
+        
+        return $bitset;
+    }
+
+    /**
+     * Set the bitset, it resets the current position of the iterator
+     * @param int $bitset
+     * @return void
+     */
+    public function setBitset($bitset) {
+        if(!is_string($bitset)) {
+            throw new \InvalidArgumentException("bitset must be a string");
+        }
+        
+        $length=ceil($this->ordinalMax/$this->integerSize)*$this->integerSize/8;
+        
+        if(strlen($bitset)<$length) { //add "\0" if the given string is not long enough
+            $bitset.=str_repeat("\0", $length-strlen($bitset));
+        }
+        
+        $this->bitset =unpack(
+            ($this->integerSize==32?'V':'P').ceil($this->ordinalMax/$this->integerSize),
+            $bitset);
+        $this->bitset=array_splice( $this->bitset,0);//splice for index shift to 0
```

Comment:
Thx haven't know that command, but now i use array_reverse because big endian export/import

Ti was because pack command return an array started with 1,2,.. not by 0,1,..